tools/apigee-x-trial-provision/apigee-x-trial-provision.sh

```diff
@@ -160,20 +192,62 @@ if [ ! "$QUIET" = "Y" ]; then
   fi
 fi
 
-export MIG=apigee-proxy-$REGION
+echo "Step 2: Enable APIs"
+gcloud services enable apigee.googleapis.com cloudresourcemanager.googleapis.com servicenetworking.googleapis.com cloudkms.googleapis.com --project="$PROJECT"
+
+echo "Step 4: Configure service networking"
+
+if [[ "$NETWORK" =~ ^projects/.*/networks.*$ ]]; then
+  set +e
+  gcloud compute networks describe "$NETWORK" --project "$PROJECT" | grep -q "$SUBNET"
+  SUBNET_FOUND=$?
+  set -e
+  if [ "$SUBNET_FOUND" = "0" ]; then
+    echo "[INFO] using existing shared VPC subnet $SUBNET"
+  else
+    echo "[FATAL] $SUBNET is not shared with this project"
```

Comment:
this logging method of `[INFO]` and `[FATAL]` isnt consistent with the rest of devrel. is it necessary? doesnt the message + exit code already give us the information? my concern is that since we arent consistently using this pattern, consumers cant filter based on these severities across devrel anyway, so it may add confusion